src/app/styles/globals.css

```diff
@@ -121,3 +121,9 @@ input[type="number"]::-ms-clear {
 .custom-auto-row-grid {
   grid-auto-rows: 0fr auto 0fr;
 }
+
+h1,
+h2,
+h3 {
+  margin: 0 !important;
```

Comment:
We want to avoid using custom classes would it be possible to do something like this?
!m-0

https://tailwindcss.com/docs/configuration#important-modifier

Reply:
This is not really a custom class, but more a global style applied to all headers. I decided to include it in global styles as I noticed I usually remove margins just to add customized margins back for all headings I use. We also have global styles for other usecases. 

But if you prefere I can either (1) remove and use !m-0 as you suggested or (2) add it to an already specified global class (see attached image):

![IMG_20240414_125926](https://github.com/DeXter-on-Radix/website/assets/44790691/90a07f3a-3fdc-49dd-a4b6-d6302a52173f)

@nguvictor

Reply:
added the heading to an already existing global style as described in my previous comment (see screenshot). LMK if you have concerns still, happy to change and use `!m-0` if you have any objections on why not to use global class for this usecase :)